nautobot/core/utils/logging.py

```diff
@@ -10,20 +10,36 @@
 
 def sanitize(string, replacement="(redacted)"):
```

Comment:
Should we rename the input arg to `dirty` or something besides string?

Reply:
I was hesitant to do so just on the off chance that some app might be explicitly calling `sanitize(string=...)` but it may be desirable.